packages/dita-example-sx-modules-xsd-topic-mod/src/configureReadOnlyElements.js

```diff
@@ -0,0 +1,29 @@
+import configureAsReadOnly from 'fontoxml-families/src/configureAsReadOnly.js';
+import configureProperties from 'fontoxml-families/src/configureProperties.js';
+import createLabelQueryWidget from 'fontoxml-families/src/createLabelQueryWidget.js';
+
+// These configurations are for testing but not for real life usages.
+export default function configureReadOnlyElements(sxModule) {
+	configureAsReadOnly(
+		sxModule,
+		`self::*[
```

Comment:
Oh wow, this is going to be expensive to compute. Heavy selector with close to no bucketing.

How would you like something like `self::p and contains(., 'READ_ONLY')`, and the same for self::simpletable and self::note or something?

Reply:
There are a lot of elements to make readonly. And when you make an element readonly according to its own content, it is not possible to revert it back, so it can be really annoying. Actually I would like to keep this commit on this feature branch.